components/EmptyListView.js

```diff
@@ -0,0 +1,38 @@
+import PropTypes from 'prop-types';
```

Comment:
let's use this new component inside FlatList directly. -> https://reactnative.dev/docs/flatlist#listemptycomponent No need to conditionally render anymore

Reply:
Great idea... That's exactly where I was going to use this.